TODO

```diff
@@ -9,6 +9,10 @@ Bugfixes:
 
 * Get rid of nftw(). We should refuse to use such useless APIs on principle.
 
+* Jun 01 09:43:02 krowka systemd[1]: Unit user@1000.service has alias user@.service.
+  Jun 01 09:43:02 krowka systemd[1]: Unit user@6.service has alias user@.service.
+  Jun 01 09:43:02 krowka systemd[1]: Unit user-runtime-dir@6.service has alias user-runtime-dir@.service.
```

Comment:
hmm, what does this mean? do you want this output? or do you want it tweaked? not following?

Reply:
That's current output. It doesn't make much sense, so I want to suppress it. I added the entry to TODO because I had other things to work on, but this is something to fix.